mcda/utils/utils_for_main.py

```diff
@@ -47,24 +47,20 @@ def check_config_error(condition: bool, error_message: str):
 
 def check_config_setting(condition_robustness_on_weights: bool,
                          condition_robustness_on_indicators: bool,
-                         mc_runs: int) -> (int, int):
+                         mc_runs: int) \
+        -> (int, int):
```

Comment:
Any specific reason to wrap this to next line?

Reply:
Nope :-D fixed.